regression/testdata/12hr80tps4org2chan-network-spec.yml

```diff
@@ -17,7 +19,7 @@ tls: true
 metrics: true
 #! true - enable gossip and dynamic leader election
 #! false - disable gossip and set all peers as org leaders
-gossipEnable: false
+gossipEnable: true
```

Comment:
Why are we enabling gossip everywhere?

Reply:
It is not enabling or disabling gossip. With false, we are making all peers as individual gossip leaders. With true, we make peers in the organization to elect leader. So considering the fact that production fabric networks will have leader election rather than making all peers as leaders, made change to have leader election enabled in all the tests.

Reply:
We actually need to revisit this, the conversation we had this morning after Scott's comment is no longer valid.

Fabric just changed the default (literally 10 minutes ago) to no longer use leader election. This is in prep for work that is going to be done on gossip.

We should set most of these to false actually (with the exception of one test that sets it to true)